### DISCUSSION TITLE
T-Beam v.1.1 to Sideband via BT.

### DISCUSSION THREAD

**Discussion by schmi1z** [markqvist/Reticulum#587] [Help & Questions] [answered]
Hello! 
I'm newbie with this projects but I have weird behavior with it... 
This application could connect with Heltec v3 node via BT but t-beam wont' to connect. I looked on BT icon on my Android and r-node but both aren't changing when Sideband is trying to connect... devices are paired :)
Maybe it is some bug with firmware for this board (I see different types of paired BT devices)... maybe the issue with "hands.sys" driver? Where I'm wrong?  :D 

thank you! 
![llg](https://github.com/user-attachments/assets/f762b6b6-2200-4ba6-a85d-362bd057e061)

**Comment by michael23505** [comment]
i'm having the same problem with rak4631. they pair to my phone, but sideband can't see them. i will be getting a display soon and will do more testing.

**Reply by michael23505** [reply]
the read display function on the web flasher was a huge help for me. i was able to figure out my specific issues that included corrupt firmware, and not initially having the bluetooth pin codes. all 5 of my rak4631's are now fully functional.

**Reply by liamcottle** [reply]
> the read display function on the web flasher was a huge help for me.

😁 I only added this as a feature a couple of days ago, glad it helped!

> all 5 of my rak4631's are now fully functional.

That is fantastic news! Nice work!

**Comment by markqvist** [comment]
You need to specify what operating system you are running this on, and how you configured Reticulum to connect, otherwise it's impossible to know what is going on.

Heltec v3 uses BLE, the T-Beam uses classic Bluetooth serial. Bluetooth connection works fine on both of these boards, but the setup procedure is different, depending on operating system.

**Reply by schmi1z** [reply]
Thank you for your answer! 
I tried to connect both boards to Android (v.13). Really, did not tried to connect them to computer via Bluetooth, but it is too interesting to check it with different OS :)

**Reply by markqvist** [reply, marked as answer]
For the Heltec v3, you must enable "Device requires BLE" in the hardware config. For the T-Beam "Device requires BLE" must be off. Both must be paired first for this to work of course.

**Reply by schmi1z** [reply]
Hello! 
Yeah, finally I was able to connect this device to my android's Sideband :) Thank you for explain :)
